Documentation/helm-values.rst

```diff
@@ -0,0 +1,1196 @@
+
+Values
+------
```

Comment:
Any chance we can get rid of this header? It doesn't bring any information that isn't on the page that includes this one, and it separates the table from its description (`The table below serves as a reference for the values that can be set on Cilium’s Helm chart.`).

Reply:
Indeed — I think I've managed to remove it :+1: